### PR TITLE
Homeworks #2 and #3

### PR DESCRIPTION
**Pull request by KamodaP**

### CODE REVIEW

File: MINI_2016/Homeworks/Third/PiotrKamoda.Rmd [hunk 1]
```diff
@@ -0,0 +1,87 @@
...
+---
+
+
+```{r setup, echo=FALSE}
+library(rpart)
+library(randomForest)
+d1=read.table("student-mat.csv",sep=";",header=TRUE)
+d1Train <- d1[0:300,]
+d1Test <- d1[301:600,]
+d1Train$Dalc <- factor(d1Train$Dalc) #Make sure setup is ok
```
Comment (awroble) [inline MINI_2016/Homeworks/Third/PiotrKamoda.Rmd:21]: please try more sophisticated way of splitting dataset, e.g. random sample

File: MINI_2016/Homeworks/Second/PiotrKamoda.Rmd [hunk 2]
```diff
@@ -0,0 +1,171 @@
...
+str(indxTrain)
+
+d1Train <- d1[indxTrain$Resample1,]
+d1Test <- d1[-indxTrain$Resample1,]
+d1Train$Dalc <- factor(d1Train$Dalc) #Make sure setup is ok
+d1Test$Dalc <- factor(d1Test$Dalc) #Make sure setup is ok
+```
+
+I've chosen Dalc as Factor variable and failures and absences as reference data. Now lets find optimal k for that:
+
```
Comment (awroble) [inline MINI_2016/Homeworks/Second/PiotrKamoda.Rmd:33]: Why do you chose these reference variables ?

Comment (KamodaP) [reply MINI_2016/Homeworks/Second/PiotrKamoda.Rmd:33]: The task didn't mention any criteria for those variables. And it seemed to me number of failures and absences could be dependent on amount of the alcohole consumed and thus provide some meaningfull results.

Comment (awroble) [reply MINI_2016/Homeworks/Second/PiotrKamoda.Rmd:33]: Please make data driven reasoning, that means show it with a data, e.g. correlation diagram that was introduced during our labs.

File: MINI_2016/Homeworks/Second/PiotrKamoda.Rmd [hunk 3]
```diff
@@ -0,0 +1,171 @@
...
+```{r tune}
```
Comment (awroble) [inline MINI_2016/Homeworks/Second/PiotrKamoda.Rmd:34]: Please normalise the input variables.

File: MINI_2016/Homeworks/Second/PiotrKamoda.Rmd (hunk 2 above)
Comment (KamodaP) [reply MINI_2016/Homeworks/Second/PiotrKamoda.Rmd:33]: I can't find in the lectures information on how to make data driven reasoning on which variables to choose.

Comment (awroble) [reply MINI_2016/Homeworks/Second/PiotrKamoda.Rmd:33]: I have told it to students as a summary of your homework, we can talk about your problems at our labs. Best

**Pull request closed without merging**